FrEee/Game/Interfaces/ICommand.cs

```diff
@@ -1,6 +1,8 @@
-﻿using FrEee.Game.Objects.Civilization;
+using FrEee.Game.Objects.Civilization;
 using System.Collections.Generic;
 
+#nullable enable

```

Comment:
So this `#nullable enable` thing you're using everywhere - is this only temporary, until everything is converted and we can enable it globally in the project file?

Reply:
Yes, this allows enabling the nullability checking on per-file basis, making the number of warnings on each step of the process manageble - there were a couple thousand when enabling null checking project-wide at first. Once I'm through the project, I will remove all the #nullable directives and enable it in the project file.